ArduPlane/Attitude.cpp

```diff
@@ -462,11 +460,15 @@ void Plane::calc_nav_yaw_coordinated(float speed_scaler)
             millis() - plane.guided_state.last_forced_rpy_ms.z < 3000) {
         commanded_rudder = plane.guided_state.forced_rpy_cd.z;
     } else {
+        if (control_mode == STABILIZE && rudder_in != 0) {
+            disable_integrator = true;
```

Comment:
might as well make it:
`bool disable_integrator = (control_mode == STABILIZE && rudder_in != 0)`